### DISCUSSION TITLE
Exodus Wallet merge - new version library release?

### DISCUSSION THREAD

**Discussion by ExodusAndy** [wevm/wagmi#601] [General]
Since Wagmi [merged the Exodus Wallet](https://github.com/tmm/wagmi/pull/583), can Wagmi release a new version of your library? Many thanks!

**Comment by tmm** [comment]
Will happen soon! Few other changes left to add to the release.

**Reply by ExodusAndy** [reply]
Thanks!

**Reply by Keith-Exodus** [reply]
Hey @tmm - wanted to check in, has the latest version been released yet? Thanks.

**Reply by tmm** [reply]
Not yet. Once this PR is merged in https://github.com/tmm/wagmi/pull/589

**Reply by Keith-Exodus** [reply]
Hey @tmm - thanks for that.

I see that the PR is now merged 🎉 

Has the latest version with Exodus being released? Thanks again.
